Tasks/RunDistributedTests/task.json

```diff
@@ -1,8 +1,8 @@
 ﻿{
     "id": "D353D6A2-E361-4A8F-8D8C-123BEBB71028",
-    "name": "RunDistributedTests",
-    "friendlyName": "Run Tests using Test Agent",
-    "description": "Run tests distributedly using multiple test agents in a lab environment. This task is under Preview, please contact devops_tools@microsoft.com to enable this task fully on the backend for your account. Please refer http://aka.ms/ci_cd for additional guidance.",
+    "name": "RunVisualStudioTestsusingTestAgent",
+    "friendlyName": "Run Visual Studio Tests using Test Agent",
+    "description": "Run tests distributedly using multiple test agents in a lab machine group.",
     "category": "Test",
     "visibility": [
                 "Preview",
```

Comment:
Jeff - Thanks for pointing out on azureConnection. These changes came into master after I pulled the branch. I did a merge to pick those changes. With regard to csproj, I don't see any csproj. I double checked.

Siddhartha - This change does not include modifications to visibility for Test tasks. They were already present in master. The edits are only to Names as you can see in the diff. I checked the visibility, it does not have Release.